h/admin.py

```diff
@@ -207,6 +207,12 @@ def badge_remove(request):
     request.db.delete(models.Blocklist.get_by_uri(uri))
     return badge_index(request)
 
+@view.view_config(route_name='admin_groups',
+                  request_method='GET',
+                  renderer='h:templates/admin/groups.html.jinja2',
+                  permission='admin_groups')
+def groups_index(request):
+    return {"groups": models.Group.all(), "request": request}
```

Comment:
You shouldn't need to pass `request` in explicitly, as it's one of the [system values provided to renderers by Pyramid](http://docs.pylonsproject.org/projects/pyramid/en/latest/narr/templates.html#renderer-system-values).